architecture/core_concepts/routes.adoc

```diff
@@ -448,28 +448,29 @@ will be used for TLS termination.
 apiVersion: v1
 kind: Route
 metadata:
-  name: route-edge-secured
+  name: route-edge-secured <1>
 spec:
   host: www.example.com
   to:
     kind: Service
-    name: service-name
+    name: service-name <1>
   tls:
-    termination: edge            <1>
-    key: |-                      <2>
+    termination: edge            <2>
+    key: |-                      <3>
       -----BEGIN PRIVATE KEY-----
       [...]
       -----END PRIVATE KEY-----
-    certificate: |-              <3>
+    certificate: |-              <4>
       -----BEGIN CERTIFICATE-----
       [...]
       -----END CERTIFICATE-----
-    caCertificate: |-            <4>
+    caCertificate: |-            <5>
       -----BEGIN CERTIFICATE-----
       [...]
       -----END CERTIFICATE-----
 ----
-<1> The `*termination*` field is `edge` for edge termination.
+<1> The name of the object, which is limited to 63 characters.
+<2> The `*termination*` field is `edge` for edge termination.
```

Comment:
Key and certificate were already swapped before your change, and you forgot to shift the numbers after 2 here, so now there are two 2's here, and the numbers down here don't match up with the numbers above.